apps/guardian/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "guardian",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc && vite build",
+        "preview": "vite preview",
+        "tauri": "tauri",
+        "lint": "eslint . --cache --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+        "lint-fix": "yarn lint --fix",
+        "prettier": "prettier . --ignore-path ../.gitignore --ignore-path ../.prettierignore --ignore-path .gitignore",
+        "fmt": "yarn prettier --write",
+        "fmt-check": "yarn prettier --check"
+    },
+    "dependencies": {
+        "@tauri-apps/api": "^1.5.2",
+        "ccd-bootstrap": "workspace:^",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router-dom": "^6.21.1"
+    },
+    "devDependencies": {
+        "@tauri-apps/cli": "^1.5.8",
+        "@types/eslint": "^8",
+        "@types/react": "^18.2.15",
+        "@types/react-dom": "^18.2.7",
+        "@vitejs/plugin-react": "^4.2.1",
+        "eslint": "^8.56.0",
+        "prettier": "^3.1.1",
+        "typescript": "^5.0.2",
+        "vite": "^5.0.0",
+        "vite-tsconfig-paths": "^4.2.3"
+    }
+}
```

Comment:
This comment is in the wrong place, but I cannot put it on the image. But can you please remove tauri.svg icon from the repo? I suppose we should use the concordium logo instead.

Reply:
Samve for  vite.svg.

Reply:
Please also replace all the icons/*.png with Concordium ones.

Tauri comes with a tool to generate the right ones from an image https://tauri.app/v1/guides/features/icons/

Reply:
thanks for the link 😄